pygmt/tests/test_grd2xyz.py

```diff
@@ -52,44 +49,6 @@ def test_grd2xyz_format(grid):
     assert list(xyz_df.columns) == ["lon", "lat", "z"]
 
 
-def test_grd2xyz_file_output(grid):
```

Comment:
The four tests are not directly related to `grd2xyz`. `test_grd2xyz_file_output` is already covered by the doctest of the `Session.virtualfile_out` method. The other three tests will be covered by doctests in PR #3098.